packages/react-ui-core/src/LeadForm/LeadForm.js

```diff
@@ -114,6 +121,7 @@ export default class LeadForm extends Component {
         {...props}
       >
         {this.fields}
+        {children}
```

Comment:
I guess we're just doing this so we can throw stuff into the form?

Reply:
Yeah... unfortunately there are things that are difficult to derive like the lead message to have it passed directly into the `fields` prop. It also allows us to add things like a wrapper around button because mobile has a wrapper div (in white) that is positioned absolute on the bottom of the viewport.

Reply:
We could also just create a custom component that does this instead, but this just gives you different options.